static/src/javascripts/projects/commercial/modules/prebid/prebid.js

```diff
@@ -28,6 +28,22 @@ const consentManagement = {
     allowAuctionWithoutConsent: true,
 };
 
+const s2sConfig = {
+    accountId: '1',
```

Comment:
What's this?

Reply:
That's our whitesocks account ID. I couldn't believe myself it was just '1', but the white guys say it is.